lib/mri/mkmf.rb

```diff
@@ -667,7 +667,8 @@ def with_ldflags(flags)
   end
 
   def try_ldflags(flags, opts = {})
-    try_link(MAIN_DOES_NOTHING, flags, {:werror => true}.update(opts))
+    # We cannot set werror => true as this produces an error if we don't use the library we are linking to.
```

Comment:
These comments need a Truffle prefix so we can find them again and understand them when we update MRI.